modules/heat_conduction/doc/content/source/kernels/JouleHeatingHeatGeneratedAux.md

```diff
@@ -0,0 +1,13 @@
+#JouleHeatingHeatGeneratedAux
```

Comment:
Create an "auxkernels" folder under source and move this file into that location.